snow/engine/avalanche/bootstrap/bootstrapper_test.go

```diff
@@ -191,31 +189,25 @@ func TestBootstrapperSingleFrontier(t *testing.T) {
 			return vtx1, nil
 		case bytes.Equal(vtxBytes, vtxBytes2):
 			return vtx2, nil
+		default:
+			t.Fatal(errParsedUnknownVertex)
```

Comment:
why not use `require.FailNow` here? (Similar in other places)

Reply:
I guess it might be nice to just ban usage of `t.Fatal` entirely at some point... so I'm fine removing it